source/reference/metrics.txt

```diff
@@ -0,0 +1,270 @@
+.. _app-services-metrics:
+
+==============================
+App Services Metrics Reference
+==============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+App Services provides a variety of metrics related to App Services and Device
+Sync. You can access your metrics data with the Admin API's
+:admin-api-endpoint:`getMetrics <tag/metrics/operation/adminGetMetrics>` endpoint.
+
+The content on this page is provided for your information. The exact metrics
+returned may be subject to change.
```

Comment:
what's the purpose of the first sentence "The content on this page is provided for your information."?

Reply:
Just a preamble to the next thing. But even that's questionable - eng says they might be private/subject to change but product would prefer they don't change.